msu/systems/mod_settings/mod_settings_system.nut

```diff
@@ -76,6 +76,12 @@
 		}
 	}
 
+	function onSettingsButtonPressed( _data )
```

Comment:
I'd go with onSettingPressed just in case we move this to AbstractSetting at some point

Reply:
Fair